app/src/main/java/org/openimis/imispolicies/domain/PolicyRenewal.java

```diff
@@ -0,0 +1,178 @@
+package org.openimis.imispolicies.domain;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.Date;
+
+public class PolicyRenewal implements Parcelable {
+
+    private final int id;
+    @NonNull
+    private final String uuid;
+    private final int policyId;
+    private final int officerId;
+    @NonNull
+    private final String officerCode;
+    @NonNull
+    private final String chfId;
+    @NonNull
+    private final String lastName;
+    @NonNull
+    private final String otherNames;
+    @NonNull
+    private final String productCode;
+    @Nullable
+    private final String productName;
+    @Nullable
+    private final String villageName;
+    @NonNull
+    private final Date renewalPromptDate;
+    @Nullable
+    private final String phone;
+
+    public PolicyRenewal(
+            int id,
+            @NonNull String uuid,
```

Comment:
if you agree with UUID type:

```suggestion
            @NonNull java.util.UUID uuid,
```